src/core/control/Control.cpp

```diff
@@ -1235,7 +1235,7 @@ void Control::manageToolbars() {
 
     this->win->updateToolbarMenu();
 
-    auto filepath = Util::getConfigFile(TOOLBAR_CONFIG);
+    auto filepath = configSearchPath->findFile(TOOLBAR_CONFIG_FILE);
     this->win->getToolbarModel()->save(filepath);
```

Comment:
If this finds a `toolbar.ini` in a non-writable system directory, it will attempt and fail to save. I think it may be better to load from the search path but save to the user's config directory.